option optimize_for = LITE_RUNTIME;

message CSPointsRaceRevengeRequest
{
 required uint32 index = 1;
}
